src/marlin_stubs/host/M115.cpp

```diff
@@ -43,9 +43,13 @@ void GcodeSuite::M115() {
     SERIAL_ECHOPGM("FIRMWARE_NAME:Prusa-Firmware-Buddy ");
     SERIAL_ECHOPGM(project_version_full);
     SERIAL_ECHOPGM(" (Github) SOURCE_CODE_URL:https://github.com/prusa3d/Prusa-Firmware-Buddy");
-#if PRINTER_IS_PRUSA_MK4
+#ifdef PRINTER_IS_PRUSA_MK4 || PRINTER_IS_PRUSA_MK4
     SERIAL_ECHOLNPGM(" PROTOCOL_VERSION:" PROTOCOL_VERSION " MACHINE_TYPE:"
+    #if PRINTER_IS_PRUSA_MK4
```

Comment:
There is a macro for printer name somewhere, use it instead of ifdefs

Reply:
Solved.